subprojects/packagefiles/libccp4c/meson.build

```diff
@@ -13,8 +13,25 @@ mdep = cc.find_library(
   required: false,
 )
 
-if host_machine.system() == 'windows'
-  error('Building libccp4c is unsupported on Windows')
+if cc.get_id() == 'msvc'
```

Comment:
Not sure how clang-cl handles this, but I would guess the same as MSVC. So use get_argument_syntax.